src/main/java/seedu/address/model/module/member/Member.java

```diff
@@ -114,8 +114,7 @@ public boolean equals(Object other) {
                 && otherMember.getPhone().equals(getPhone())
                 && otherMember.getEmail().equals(getEmail())
                 && otherMember.getAddress().equals(getAddress())
-                && otherMember.getPositions().equals(getPositions())
-                && otherMember.getTaskList().equals(getTaskList());
+                && otherMember.getPositions().equals(getPositions());
```

Comment:
I am not sure whether shall we delete `otherMember.getTaskList().equals(getTaskList());`